docs/understanding-tracking-design/create-tracking-plan-with-tracking-scenarios/index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Creating a Tracking Plan with Tracking Scenarios"
+sidebar_label: "🆕 Creating a Tracking Plan"
+sidebar_position: 100
+sidebar_custom_props:
+  offerings:
+    - enterprise
+---
+
+As explained in [Introduction to Tracking Design](/docs/understanding-tracking-design/introduction-to-tracking-design/index.md), to use Snowplow successfully, you need to have a good idea of:
+- The events that matter to your business
+- The events that occur in your website, mobile application, server-side systems, factories, call centers, dispatch centers, etc.
+- The decisions you make based on those events
+- What you need to know about those events to make those decisions
+
+While our Schema technology and Data Structures UI can help you capture the structure and instructions for validating your data, we are now introducing Tracking Scenarios to help capture additional information, such as:
```

Comment:
Make this a link to `/docs/understanding-your-pipeline/schemas/index.md`

Reply:
Done